websites/S/Sosac/metadata.json

```diff
@@ -36,4 +36,4 @@
 	],
 	"iframe": true,
 	"iFrameRegExp": "www[.]streamuj[.]tv[/]video[/]"
-}
+}
```

Comment:
Undo changes to the file

Reply:
Why does it keep changing it? I'm only changing `/websites/V/VtuberWiki`

Reply:
Its not even there.

![image](https://github.com/PreMiD/Presences/assets/93791569/b1a3ffbc-a4bf-4be6-b56d-0989a28b38c4)
![image](https://github.com/PreMiD/Presences/assets/93791569/468dca41-0350-4ca1-b57e-98bf21ae5118)